en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -55,13 +55,13 @@ Coordinate X: 1
 
 This lesson assumes you will be using a single Node console session, unless otherwise specified.
 
-You can keep the same session logged on for most of the examples, but in the case that you find anything unexpected, you should try restarting the console before doing anything else. In order to restart the console, you have to **exit and enter again.**
+You can keep the same session logged on for most of the examples, but in the case that you find anything unexpected, you should try restarting the console before doing anything else. In order to restart the console, you have to **exit and enter again**.
 
 It would be best for you to keep this text opened up in a window (or printed out,) and the Node console on another.
 
 ## Primitive types
 
-**Primitive types** is how we call types that come built into the language and allow for the creation of newer, more complex types. The primitive types in JavaScript are: **boolean**, **number**, **string**, **object**, and **function.**

```

Comment:
I'm not sure about this change. "Primitive types" in this case is just a name we are referring to - THE way to designate a category, hence the singular. I do realize in this case it should go between quotation marks or maybe be rephrased into something like "Primitive types is the name we give to ...", but they weren't there in the original. This is a change I'll probably make after merging your proposed changes.